lib/components/markdown.js

```diff
@@ -0,0 +1,20 @@
+import MarkdownIt from 'markdown-it'
+
+const md = new MarkdownIt('default', {
+  html: true,
+  linkify: true,
+  breaks: false,
+})
+
+const defaultRender = md.renderer.rules.link_open ||
+  function (tokens, idx, options, env, self) {
+    return self.renderToken(tokens, idx, options)
+  }
+
+md.renderer.rules.link_open = function (tokens, idx, options, env, self) {
+  tokens[idx].attrJoin('rel', 'noopener noreferrer ugc')
+
+  return defaultRender(tokens, idx, options, env, self)
+}
+
+export default md
```

Comment:
This PR uses an outdated markdown and XSS parser. See the master branch (it has been updated since nuxt 3) and update it